src/lsm/manifest_log.zig

```diff
@@ -266,9 +237,9 @@ pub fn ManifestLogType(comptime Storage: type, comptime TableInfo: type) type {
             const block_reference = manifest_log.read_block_reference.?;
             verify_block(block, block_reference.checksum, block_reference.address);
 
-            const entry_count = Block.entry_count(block);
-            const labels_used = Block.labels_const(block)[0..entry_count];
-            const tables_used = Block.tables_const(block)[0..entry_count];
+            const entry_count = schema.ManifestLog.entry_count(block);
+            const labels_used = schema.ManifestLog.labels_const(block)[0..entry_count];
+            const tables_used = schema.ManifestLog.tables_const(block)[0..entry_count];
```

Comment:
:thinking: I don't love `buffer` since it implies to me that we are getting back bytes, not tables/labels/etc What about `_free`?

Reply:
🤔 ”getting back bytes” isn’t exactly wrong here, because what we are getting here is raw memory indeed, it doesn’t necessary contain *valid* tables.

free doesn’t feel right, because *some* entries might not be free.